### ISSUE TITLE
RAZER KRAKEN KITTY V2

### ISSUE THREAD

**Issue by mhiecaaa** [stickoking/razer-macos#8]
hi please make it available to my headsetttt. coz its not working on razer macos. my mac recogizes my headset.

**Comment by mhiecaaa** [comment]
hello please make it availableeee

Razer Kraken Kitty V2:

  Product ID:	0x0560
  Vendor ID:	0x1532
  Version:	10.06
  Serial Number:	00000000
  Speed:	Up to 12 Mb/s
  Manufacturer:	Razer
  Location ID:	0x14200000 / 9
  Current Available (mA):	500
  Current Required (mA):	100

**Comment by stickoking** [comment]
Hello I will add support please give some time

**Comment by mhiecaaa** [comment]
Thank you ill patiently wait

**Comment by metafarion** [comment]
Yes please.  How am I supposed to look cute in my meetings if my ears don't change color?
